platform/viewer/src/googleCloud/api/GoogleCloudApi.js

```diff
@@ -15,7 +15,7 @@ class GoogleCloudApi {
   }
 
   get urlBase() {
-    return `https://healthcare.googleapis.com/v1beta1`;
+    return window.config.healthcareApiEndpoint;
```

Comment:
Can you add || 'https://healthcare.googleapis.com/v1beta1' here?

Thanks!

Reply:
This expands our appContext and configuration. There may be a cleaer way to pull this value after the app's been kickstarted.

Reply:
> Can you add || 'https://healthcare.googleapis.com/v1beta1' here?
Done.

Reply:
> This expands our appContext and configuration. There may be a cleaer way to pull this value after the app's been kickstarted.

Probably `getServers()` in the `platform/viewer/src/customHooks/useServer.js` is the most suitable place. What do you think?